packages/actions/src/Exports/Jobs/PrepareCsvExport.php

```diff
@@ -69,13 +69,18 @@ public function handle(): void
 
         if ($databaseConnection->getDriverName() === 'pgsql') {
             $originalOrders = collect($query->getQuery()->orders)
-                ->reject(fn (array $order): bool => in_array($order['column'] ?? null, [$keyName, $qualifiedKeyName]))
```

Comment:
Can you please explain the removal of the non-qualified key name from this array? That is the part I am most unsure about in this PR

Reply:
This is because sorting in DESC order causes an issue when the table has an id column.
line 82`$query->reorder($qualifiedKeyName);` does not specify a sort direction, so it always defaults to ASC.

![スクリーンショット 2024-12-31 141555](https://github.com/user-attachments/assets/8418e24e-5727-43de-b2af-c58bc909bc04)

result: 
```csv
ID,DATA_OF_BIRTH,NAME,TYPE,HAS_RECOVERED
100,2016-11-21,Dixie,cat,false
99,2018-11-29,Arturo,cat,true
...
2,2019-04-27,Pattie,rabbit,false
1,2018-07-02,Krystal,dog,true
150,2019-04-27,Sydnee,dog,false
149,2019-12-13,Leonel,rabbit,false
```